commit cc4c9b2260278d8bbc67e0346fc23ce7549fc1fc
Author: Jens Lody <jens@codeblocks.org>
Date:   Wed Jul 25 15:17:21 2012 +0200

    * wxSTC: speed up scrolling of small amount of lines a lot

Index: src/stc/ScintillaWX.cpp
===================================================================
--- src/stc/ScintillaWX.cpp
+++ src/stc/ScintillaWX.cpp
@@ -364,7 +364,6 @@ bool ScintillaWX::HaveMouseCapture() {
 void ScintillaWX::ScrollText(int linesToMove) {
     int dy = vs.lineHeight * (linesToMove);
     stc->ScrollWindow(0, dy);
-    stc->Update();
 }
 
 void ScintillaWX::SetVerticalScrollPos() {
